src/NetscriptBladeburner.js

```diff
@@ -0,0 +1,27 @@
+import {Player}               from "./Player";
+import {Bladeburner}          from "./Bladeburner";
+import {makeRuntimeRejectMsg} from "./NetscriptEvaluator";
+
+function unknownBladeburnerActionErrorMessage(functionName, actionType, actionName) {
+	return `ERROR: bladeburner.${functionName}() failed due to an invalid action specified. ` +
+		`Type: ${actionType}, Name: ${actionName}. Note that for contracts and operations, the ` +
+		`name of the operation is case-sensitive.`;
+}
+
+function unknownBladeburnerExceptionMessage(functionName, err) {
+	return `Bladeburner.${functionName}() failed with exception: ` + err;
+}
+
+function checkBladeburnerAccess(workerScript, functionName) {
+	const accessDenied = `${functionName}() failed because you do not` +
+		" currently have access to the Bladeburner API. This is either" +
+		" because you are not currently employed at the Bladeburner division" +
+		" or because you do not have Source-File 7";
+	console.log(Player.sourceFiles);
```

Comment:
Probably shouldn't include the console logging if we can help it. Or at the very least it should be `console.debug`

Reply:
I imagine this was accidentally left in. I'll remove it